bundler.d/assets.rb

```diff
@@ -1,7 +1,7 @@
 group :assets do
   gem 'sass-rails', '~> 3.2'
   gem 'uglifier', '>= 1.0.3'
-  gem 'execjs', '>= 1.4.0'
+  gem 'execjs', '>= 1.4.0', '<2.5.0'
```

Comment:
I'm very sure if anything arises, execjs people will tell that 2.5.0 is 2.0 only.